gui/src/app/StanSampler/StanSampler.ts

```diff
@@ -58,7 +72,15 @@ class StanSampler {
         }
         this.#worker.postMessage({ purpose: Requests.Load, url: this.compiledUrl });
     }
-    sample(sampleConfig: Partial<SamplerParams>) {
+    sample(data: string | object, samplingOpts: SamplingOpts) {
+        const sampleConfig: SamplerParams = {
```

Comment:
If you keep the mains consistent as you have thus far, this can be

```
        const sampleConfig: Partial<SamplerParams> = {
            data,
            ...samplingOpts
        }
```

Which I think also fixes the issue I'm having where it's trying to import `defaultSamplerParams` from tinystan which is private

Reply:
Yes I just fixed that, thanks.